### PR TITLE
test: get decision definition xml - integration tests

### PR DESCRIPTION
**Pull request by houssain-barouni**
## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

relates to https://github.com/camunda/camunda/issues/20654

### CODE REVIEW

File: qa/integration-tests/src/test/java/io/camunda/it/client/GetDecisionDefinitionXmlTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import java.util.List;
+import java.util.Objects;
+import java.util.Random;
+import java.util.concurrent.CompletionException;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+@ZeebeIntegration
+public class GetDecisionDefinitionXmlTest {
```
Comment (tmetzke) [inline qa/integration-tests/src/test/java/io/camunda/it/client/GetDecisionDefinitionXmlTest.java:34]: ❓ Just wondering: should we maybe integrate the test cases into the `SearchDecisionDefinitionTest` and rename that one? Could help us save resources used for integration tests. Just a thought, let me know what you think.

Comment (houssain-barouni) [reply qa/integration-tests/src/test/java/io/camunda/it/client/GetDecisionDefinitionXmlTest.java:34]: That makes sense; there’s no need to launch a new test application and Elasticsearch instance just for these couple of tests.

Comment (houssain-barouni) [reply qa/integration-tests/src/test/java/io/camunda/it/client/GetDecisionDefinitionXmlTest.java]: I moved the tests to `DecisionQueryTest` I also did the same for `SearchDecisionRequirementsTest` tests which share the same preconditions

**Review by tmetzke — APPROVED** [review]
👍 Great job, I really like the new test coverage and that we aggregate decision tests to reuse test resources 🚀

**Pull request merged**